target/linux/ath79/image/generic.mk

```diff
@@ -767,6 +767,15 @@ define Device/devolo_dlan-pro-1200plus-ac
 endef
 TARGET_DEVICES += devolo_dlan-pro-1200plus-ac
 
+define Device/devolo_dlan-pro-500-wp
+  SOC := ar9344
+  DEVICE_VENDOR := Devolo
+  DEVICE_MODEL := dLAN pro wireless 500+
+  DEVICE_PACKAGES := kmod-ath9k
```

Comment:
drop `DEVICE_PACKAGES := kmod-ath9k`. The default package already contains `kmod-ath9k`.